src/PIL/__init__.py

```diff
@@ -83,3 +83,11 @@ class UnidentifiedImageError(OSError):
     """
 
     pass
+
+
+def __getattr__(name: str):
```

Comment:
What do you think about putting `ImageType` in `Image.py` to avoid this? In other files, would importing from `Image` instead of from `PIL` (or `.`) be tricky?

It's just a thought, I'm fine with this.

Reply:
Moving it to `Image.py` would make it easier to find in the docs ("Reference > Image module" instead of the current "Reference > PIL Package (autodoc of remaining modules)").

That said, I have no preference on putting it in either file either.

Reply:
So, I was confused by your initial suggestion of [`ImageType: TypeAlias = Image.Image` in Image.py](https://github.com/python-pillow/Pillow/pull/7635#pullrequestreview-1795553042), because if it's in Image.py, then it's not `ImageType = Image.Image`, it's just `ImageType = Image`, right? It's https://github.com/radarhere/Pillow/commit/75afb580d71146fdb8c3294d4939116214732635, yeah?

I think the users in #6676 basically don't like that `Image.Image` exists as a class within a module, and would like both to be at the package level. I think this version still not that, but closer to what they want.

I'm happy to go with either this version or the `ImageType = Image` version, since both are backwards compatible - or neither. If the goal of the type alias is brevity though, then this is the shortest version?

Reply:
Yep, this PR is better than https://github.com/radarhere/Pillow/commit/75afb580d71146fdb8c3294d4939116214732635, thanks for the illustration :)